packages/frontend/src/components/fullscreen-modal/index.tsx

```diff
@@ -36,9 +38,19 @@ export const AnimatedFullscreenModal = ({
     onDismiss,
     children,
 }: FullscreenModalProps) => {
+    const dispatch = useDispatch();
+
+    useEffect(() => {
+        dispatch(setModalOpen({ open: true }));
```

Comment:
With the suggested change below, this would become `setModalOpen(true)` and `setModalOpen(false)`